### DISCUSSION TITLE
Auto Complete

### DISCUSSION THREAD

**Discussion by BrettKnoss** [fonsp/Pluto.jl#1430] [Questions about using Pluto] [answered]
I saw on YouTube (JuliaCon 2021), that Pluto has an auto complete.  How do I get this to work?

**Comment by fonsp** [comment, marked as answer]
Use `Tab` or `Ctrl+Space` or `Cmd+Space` while typing.

(Use `Ctrl+/` or `Cmd+/` to see the list of all keyboard shortcuts.)

**Reply by BrettKnoss** [reply]
Tab brings up variables and functions.  It would be nice if it also brought up dictionary keys, unless it does and I have my dictionary set up wrong.

Ctrl+/ brings up the menue to minimize or maxize for Vivaldi, and Cmd + / brings up search on Safari.  I don't have a Cmd key on my Debian machine, so I can't test that.
